src/views/Login.jsx

```diff
@@ -0,0 +1,47 @@
+//ToDo make Login look good. UI. 
+
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import useAuth from "../contexts/AuthContextProvider";
+
+const Login = () => {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState('');
+    const { user, setUser } = useAuth();
```

Comment:
In more than 4 states, you might need to refactor this into useReducer to make it cleaner